testdata/p4_16_errors/psa-type-hdr.p4

```diff
@@ -0,0 +1,143 @@
+/*
```

Comment:
why add such a large program for a test that fails right away?

Reply:
It's a PSA 1.1 program which has so many control block which are all empty.  The ipv4_t header is needed for the test.  I can only trim down a few more lines - will look into it.

Reply:
I am guessing Mihai is thinking that test programs that are run only through p4test, and not targeted for running on simple_switch, need not be "complete" programs for any particular architecture.  There are a lot of test programs that are not complete 'switch architecture' programs.

Reply:
Thanks, Andy.  I just checked in this modified P4 program which is very small.  I also took care of two review comments in p4RuntimeSerializer.cpp.